dali/operators/image/remap/warp_affine_params.h

```diff
@@ -67,7 +73,7 @@ class WarpAffineParamProvider
       for (int i = 0; i < spatial_ndim; i++)
         for (int j = 0; j < spatial_ndim+1; j++, k++)
           M.transform(i, j) = matrix[k];
-
+      if (invert) M = M.invert();
```

Comment:
One-line ifs would be acceptable if there was a ladder of these, but here it serves as a camouflage to the conditional statement.
```suggestion
      if (invert)
        M = M.invert();
```

Reply:
done